tests/test_ssim.py

```diff
@@ -78,6 +78,22 @@ def test_ssim_raises_if_wrong_kernel_size_is_passed(prediction: torch.Tensor, ta
 
 
 # ================== Test class: `SSIMLoss` ==================
+def test_SSIM_measure_is_less_or_equal_to_one() -> None:
```

Comment:
Duplicates `test_ssim_loss_is_less_or_equal_to_one` (cuda as well). Sorry for not specifying explicitly, thought you would check for all duplicates. This is the last one.